DREAM3DReviewFilters/SurfaceMeshToWaveFront.h

```diff
@@ -0,0 +1,182 @@
+  static Pointer NullPointer();
+
+  static std::shared_ptr<SurfaceMeshToWaveFront> New();
+
+  /**
+   * @brief Returns the name of the class for SurfaceMeshToWaveFront
+   */
+  QString getNameOfClass() const override;
+
+  /**
+   * @brief Returns the name of the class for SurfaceMeshToWaveFront
+   */
+  static QString ClassName();
+
+  ~SurfaceMeshToWaveFront() override;
+
+  /**
+   * @brief Setter property for OutputWaveFrontFile
+   */
+  void setOutputWaveFrontFile(const QString& value);
+  /**
+   * @brief Getter property for OutputWaveFrontFile
+   * @return Value of OutputWaveFrontFile
+   */
+  QString getOutputWaveFrontFile() const;
+  Q_PROPERTY(QString OutputWaveFrontFile READ getOutputWaveFrontFile WRITE setOutputWaveFrontFile)
+
+  /**
+   * @brief Setter property for SurfaceMeshNodeTypeArrayPath
+   */
+  void setSurfaceMeshNodeTypeArrayPath(const DataArrayPath& value);
+  /**
+   * @brief Getter property for SurfaceMeshNodeTypeArrayPath
+   * @return Value of SurfaceMeshNodeTypeArrayPath
+   */
+  DataArrayPath getSurfaceMeshNodeTypeArrayPath() const;
+  Q_PROPERTY(DataArrayPath SurfaceMeshNodeTypeArrayPath READ getSurfaceMeshNodeTypeArrayPath WRITE setSurfaceMeshNodeTypeArrayPath)
+
+  /**
+   * @brief getCompiledLibraryName Reimplemented from @see AbstractFilter class
+   */
+  QString getCompiledLibraryName() const override;
+
+  /**
+   * @brief getBrandingString Returns the branding string for the filter, which is a tag
+   * used to denote the filter's association with specific plugins
+   * @return Branding string
+   */
+  QString getBrandingString() const override;
+
+  /**
+   * @brief getFilterVersion Returns a version string for this filter. Default
+   * value is an empty string.
+   * @return
+   */
+  QString getFilterVersion() const override;
+
+  /**
+   * @brief newFilterInstance Reimplemented from @see AbstractFilter class
+   */
+  AbstractFilter::Pointer newFilterInstance(bool copyFilterParameters) const override;
+
+  /**
+   * @brief getGroupName Reimplemented from @see AbstractFilter class
+   */
+  QString getGroupName() const override;
+
+  /**
+   * @brief getSubGroupName Reimplemented from @see AbstractFilter class
+   */
+  QString getSubGroupName() const override;
+
+  /**
+   * @brief getUuid Return the unique identifier for this filter.
+   * @return A QUuid object.
+   */
+  QUuid getUuid() const override;
+
+  /**
+   * @brief getHumanLabel Reimplemented from @see AbstractFilter class
+   */
+  QString getHumanLabel() const override;
+
+  /**
+   * @brief setupFilterParameters Reimplemented from @see AbstractFilter class
+   */
+  void setupFilterParameters() override;
+
+  /**
+   * @brief execute Reimplemented from @see AbstractFilter class
+   */
+  void execute() override;
+
+protected:
+  SurfaceMeshToWaveFront();
+
+  /**
+   * @brief dataCheck Checks for the appropriate parameter values and availability of arrays
+   */
+  void dataCheck();
```

Comment:
```suggestion
  void dataCheck() override;
```
Changed to match changes in AbstractFilter.